drivers/overlay/ov_network.go

```diff
@@ -291,7 +377,19 @@ func (n *network) KeyPrefix() []string {
 }
 
 func (n *network) Value() []byte {
-	b, err := json.Marshal(n.vxlanID())
+	overlayNetmap := make(map[string]interface{})
+
+	s := n.subnets[0]
```

Comment:
Yes, I will make the change to handle multiple subnets in a separate PR. Only the marshal/unmarshal needs some changes.